README.md

```diff
@@ -98,5 +61,10 @@ See full copyrights in [LICENSE.md][license.md] inside repository
 
 [license.md]: /LICENSE.md
 [changelog.md]: /CHANGELOG.md
-[version-explanation.png]: /Doc/version-explanation.png
 [jillejr.newtonsoft.json-for-unity]: https://github.com/jilleJr/Newtonsoft.Json-for-Unity#readme
+[wiki-Install-Converters-via-Git-in-UPM]: https://github.com/jilleJr/Newtonsoft.Json-for-Unity/wiki/Install-Converters-via-Git-in-UPM
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[definition-case] Do not use upper-case characters in definition labels](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5461644)